test/harvest/test_pubmed.py

```diff
@@ -181,6 +331,101 @@ def test_harvest_when_doi_exists(
         assert pub.authors[1].orcid == "https://orcid.org/0000-0000-0000-0002"
 
 
+def test_fill_in(snapshot, test_session, mock_publication, caplog, monkeypatch):
+    caplog.set_level(logging.INFO)
```

Comment:
Do we need this? Maybe it's left over from some testing?

Reply:
maybe?  this was in the other fill in tests, my assumption was it is there to ensure the log file level is set in a way that the assertions below would definitely work (i.e. if the default log level is changed elsewhere at some point, the assertions might start failing?)